src/main/kotlin/com/hibob/academy/employeeFeedback/dao/EmployeeDao.kt

```diff
@@ -20,4 +19,15 @@ class EmployeeDao(private val sql: DSLContext) {
             .fetchOne()!!
             .get(employeeTable.id)
 
+
+    fun getRole(userLoggedInDetails: UserLoggedInDetails): String? {
```

Comment:
Isn't Role an enum? if so - return that enum. and Role should be mandatory, right? if you got to this method in the dao, the user exists and you should return Role and not String?